### PR TITLE
Fxa 5207 verification reminder second copy

### PR DESCRIPTION
**Pull request by millsoper**
## Because

- We're updating copy for emails

## This pull request

- Updates copy, localization ids, and tests for the verificationReminderSecond email

## Issue that this pull request solves

Closes: # n/a

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Screen Shot 2022-08-10 at 2 15 40 PM

### CODE REVIEW

File: packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl [hunk 1]
```diff
@@ -1,5 +1,6 @@
-verificationReminderSecond-subject = Final reminder: Activate your account
-verificationReminderSecond-title = Still there?
-verificationReminderSecond-description = Almost a week ago you created a { -product-firefox-account } but never verified it. We’re worried about you.
+verificationReminderSecond-subject-2 = Remember to confirm your account
+verificationReminderSecond-title-2 = Don't miss out on { -brand-firefox }
+verificationReminderSecond-description-3 = A few days ago you created a { -product-firefox-account }, but never confirmed it. Please confirm your account in the next 10 days or it will be automatically deleted.
+verificationReminderSecond-second-description = Your { - product-firefox-account} lets you sync your info across devices and unlocks access to more privacy-protecting products from Mozilla.
```
Comment (bcolsson) [inline packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl:4]:
```suggestion
verificationReminderSecond-second-description = Your { - product-firefox-account } lets you sync your info across devices and unlocks access to more privacy-protecting products from { -brand-mozilla }.
```

**Review by bcolsson — CHANGES_REQUESTED** [review]

File: packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl (hunk 1 above)
Comment (millsoper) [reply packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl:4]: Will do, thank you!

Comment (bcolsson) [reply packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl]: Thanks for adding the placeholder! I'm just noticing this now but this placeholder is a bit wonky for verificationReminderSecond-second-description: 
`{ - product-firefox-account}`
should be:
`{ -product-firefox-account }`

There's a space in-between the dash and the name, and a space missing before the final bracket.

Comment (millsoper) [reply packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl]: Thank you for catching that! Updating now

File: packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/index.txt [hunk 2]
```diff
@@ -1,10 +1,12 @@
-verificationReminderSecond-title = "Still there?"
+verificationReminderSecond-title-2 = "Don't miss out on Firefox"
```
Comment (pdehaan) [inline packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/index.txt:1]:
```suggestion
verificationReminderSecond-title-2 = "Don’t miss out on Firefox"
```

File: packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/index.txt [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
 
-verificationReminderSecond-description-2 = "Almost a week ago you created a Firefox account but never confirmed it. We’re worried about you."
+verificationReminderSecond-description-3 = "A few days ago you created a Firefox account, but never confirmed it. Please confirm your account in the next 10 days or it will be automatically deleted."
 
-verificationReminderSecond-sub-description = "Confirm this email address to activate your account and let us know you’re okay."
+verificationReminderSecond-second-description = "Your Firefox account lets you sync your info across devices and unlocks access to more privacy-protecting products from Mozilla."
 
-confirm-email-plaintext = "Confirm email:"
+verificationReminderSecond-sub-description = "Be part of our mission to transform the internet into a place that's open for everyone."
```
Comment (pdehaan) [inline packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/index.txt:7]:
```suggestion
verificationReminderSecond-sub-description = "Be part of our mission to transform the internet into a place that’s open for everyone."
```

File: packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl [hunk 4]
```diff
@@ -1,5 +1,6 @@
-verificationReminderSecond-subject = Final reminder: Activate your account
-verificationReminderSecond-title = Still there?
-verificationReminderSecond-description-2 = Almost a week ago you created a { -product-firefox-account } but never confirmed it. We’re worried about you.
+verificationReminderSecond-subject-2 = Remember to confirm your account
+verificationReminderSecond-title-2 = Don't miss out on { -brand-firefox }
```
Comment (pdehaan) [inline packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl:2]:
```suggestion
verificationReminderSecond-title-2 = Don’t miss out on { -brand-firefox }
```

Comment (bcolsson) [reply packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl:2]: Thanks @pdehaan for catching the typographic quotes.

File: packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl [hunk 5]
```diff
@@ -1,5 +1,6 @@
-verificationReminderSecond-subject = Final reminder: Activate your account
-verificationReminderSecond-title = Still there?
-verificationReminderSecond-description-2 = Almost a week ago you created a { -product-firefox-account } but never confirmed it. We’re worried about you.
+verificationReminderSecond-subject-2 = Remember to confirm your account
+verificationReminderSecond-title-2 = Don’t miss out on { -brand-firefox }
+verificationReminderSecond-description-3 = A few days ago you created a { -product-firefox-account }, but never confirmed it. Please confirm your account in the next 10 days or it will be automatically deleted.
+verificationReminderSecond-second-description = Your { -product-firefox-account } lets you sync your info across devices and unlocks access to more privacy-protecting products from { -brand-mozilla }.
 verificationReminderSecond-sub-description = Confirm this email address to activate your account and let us know you’re okay.
```
Comment (xlisachan) [inline packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl:5]: This line can be deleted if it's no longer a part of the email.

File: packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/index.txt [hunk 6]
```diff
@@ -1,10 +1,12 @@
...
+verificationReminderSecond-title-2 = "Don’t miss out on Firefox"
 
-verificationReminderSecond-description-2 = "Almost a week ago you created a Firefox account but never confirmed it. We’re worried about you."
+verificationReminderSecond-description-3 = "A few days ago you created a Firefox account, but never confirmed it. Please confirm your account in the next 10 days or it will be automatically deleted."
 
-verificationReminderSecond-sub-description = "Confirm this email address to activate your account and let us know you’re okay."
+verificationReminderSecond-second-description = "Your Firefox account lets you sync your info across devices and unlocks access to more privacy-protecting products from Mozilla."
 
-confirm-email-plaintext = "Confirm email:"
+verificationReminderSecond-sub-description = "Be part of our mission to transform the internet into a place that’s open for everyone."
```
Comment (xlisachan) [inline packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/index.txt:7]: `verificationReminderSecond-sub-description-2`

Comment (millsoper) [reply packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/index.txt:7]: thank you!!

File: packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl [hunk 7]
```diff
@@ -1,5 +1,6 @@
-verificationReminderSecond-subject = Final reminder: Activate your account
-verificationReminderSecond-title = Still there?
-verificationReminderSecond-description-2 = Almost a week ago you created a { -product-firefox-account } but never confirmed it. We’re worried about you.
-verificationReminderSecond-sub-description = Confirm this email address to activate your account and let us know you’re okay.
-verificationReminderSecond-action = Confirm email
+verificationReminderSecond-subject-2 = Remember to confirm your account
+verificationReminderSecond-title-2 = Don’t miss out on { -brand-firefox }
+verificationReminderSecond-description-3 = A few days ago you created a { -product-firefox-account }, but never confirmed it. Please confirm your account in the next 10 days or it will be automatically deleted.
+verificationReminderSecond-second-description = Your { -product-firefox-account } lets you sync your info across devices and unlocks access to more privacy-protecting products from { -brand-mozilla }.
+verificationReminderSecond-sub-description-2 = Be part of our mission to transform the internet into a place that's open for everyone.
```
Comment (xlisachan) [inline packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl:5]: Sorry I missed this before - smart apostrophe is needed in "that's".

**Review by xlisachan — APPROVED** [review]
LGTM - just got to update that apostrophe in `en.ftl`.

File: packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl (hunk 7 above)
Comment (millsoper) [reply packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl:5]: Thank you!!!!

**Review by bcolsson — APPROVED** [review]

**Pull request merged**